src/main/java/entity/cards/VileWeave.java

```diff
@@ -0,0 +1,49 @@
+package entity.cards;
+
+import static entity.EntityMod.makeCardPath;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import entity.EntityMod;
+import entity.characters.Entity;
+import entity.powers.FluxPower;
+//Vile weave	common	skill	1	Block 10(13). apply 1 flux to yourself
+public class VileWeave extends AbstractDynamicCard {
+    public static final String ID = EntityMod.makeID(VileWeave.class.getSimpleName());
+    public static final String IMG = makeCardPath("AetherForm.png");
+
+    private static final CardRarity RARITY = CardRarity.COMMON;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Entity.Enums.COLOR_TEAL;
+
+    private static final int COST = 1;
+
+    private static final int BLOCK = 10;
+    private static final int UPGRADE_PLUS_BLOCK = 3;
+
+    private static final int SELF_MAGIC_NUMBER = 1;
+
+    public VileWeave() {
+        super(ID, IMG, COST, TYPE, COLOR, RARITY, TARGET);
+        this.block = this.baseBlock = BLOCK;
+        this.selfMagicNumber = this.baseSelfMagicNumber = SELF_MAGIC_NUMBER;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        AbstractDungeon.actionManager.addToBottom(new GainBlockAction(p, p, this.block));
+        AbstractDungeon.actionManager.addToBottom(new ApplyPowerAction(p, p, new FluxPower(p, p, flux), flux));
```

Comment:
This needs to use selfMagicNumber instead of flux.
```suggestion
        AbstractDungeon.actionManager.addToBottom(new ApplyPowerAction(p, p, new FluxPower(p, p, selfMagicNumber), selfMagicNumber));
```